spec/controllers/hyrax/generic_works_controller_spec.rb

```diff
@@ -351,18 +351,20 @@
           before do
             allow(work).to receive(:file_sets).and_return(double(present?: true))
           end
+
           it 'updates the work' do
             patch :update, params: { id: work, generic_work: {} }
             expect(response).to redirect_to main_app.hyrax_generic_work_path(work, locale: 'en')
           end
         end
 
+        # TODO: this test should move to the ModelActor spec
         context 'when members are set' do
           let(:file_set) { create_for_repository(:file_set) }
 
           it 'can update file membership' do
             patch :update, params: { id: work, generic_work: { member_ids: [file_set.id.to_s] } }
```

Comment:
I am seeing a failure here:
```     ArgumentError:
       id must be a Valkyrie::ID
     # ./app/services/hyrax/queries.rb:43:in `find_members'
     # ./app/services/hyrax/queries.rb:8:in `find_members'
     # ./app/controllers/concerns/hyrax/works_controller_behavior.rb:258:in `after_update_success'
     # ./app/controllers/concerns/hyrax/works_controller_behavior.rb:42:in `update'
     # ./spec/controllers/hyrax/generic_works_controller_spec.rb:366:in `block (6 levels) in <top (required)>'
```

Reply:
Works for me:
```
$ ber spec/controllers/hyrax/generic_works_controller_spec.rb

Randomized with seed 2709
.....................................
Finished in 4.98 seconds (files took 10.36 seconds to load)
37 examples, 0 failures
```
Sorry, I know that's crummy, but that's the best I can do until we get the full build done.

Reply:
yep